.github/workflows/test_manylinux.yml

```
@@ -0,0 +1,74 @@
+name: manylinux-compatibility
+
+# Ensure that cylc-flow remains installable with manylinx1 binaries for use
+# with older systems.
+
+# Ideally we would use the Docker images from PyPa
+# (https://github.com/pypa/manylinux).
+# The manylinux1 image is based on CentOS=5, glibc=2.5), however, this is so
+# old that installing contemporary Python on them is quite hard:
+# * native package manager has no package for CentOS5
+# * miniforge can't work with glibc < 2.12
+# * micromamba can't work with glibc < 2.6
+# * need zlib to compile Python / install pip but the package manager can't
+#   download it because the package stream is offline (past EOL)
+
+# So this will have to do. This isn't a perfect test as non manylinux compat
+# packages may attempt to compile from source. This might succeed here but not
+# on an older host.
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - 'setup.py'
+      - 'setup.cfg'
+
+jobs:
+  test:
```

Worth having a timeout on the whole job in case something else gets stuck. 
```suggestion
  test:
    timeout-minutes: 35
```